### ISSUE TITLE
Parse link headers (rfc5988)

### ISSUE THREAD

**Issue by daniel-j-h** [aio-libs/aiohttp#2948]
See https://github.com/requests/requests/issues/711 for context.

It would be great if aiohttp could parse link headers helping e.g. with pagination.

At the moment the [`ClientResponse` has a `headers` attribute](https://aiohttp.readthedocs.io/en/stable/client_reference.html#aiohttp.ClientResponse.headers) from which we can get the link header:

```python
print(response.headers.get('Link'))

<https://example.com//page/3>; rel="next", <https://example.com/page/1>; rel="prev"
```

But it's of type `str` and not parsed. To use the url e.g. in the `next` rel (for pagination) users currently have to parse the link header string manually pulling out the url.

**Comment by asvetlov** [comment]
The attached issue has no design for the API, only many good wishes.

Would you elaborate your vision?

**Comment by daniel-j-h** [comment]
I'm not too deep into aiohttp - I was just using it for a small app and found out about the missing link header parsing; so I don't have clear real vision. The API I am familiar with is the `requests` API for link headers:

http://docs.python-requests.org/en/master/user/advanced/#link-headers

I think we need to make two calls here:
1. Should `aiohttp` do link header parsing or should users use an external library for it?
2. Should `aiohttp` replicate the `requests` API for link headers?

**Comment by asvetlov** [comment]
Well, supporting `response.links['name']` sounds a good addition.

**Comment by asvetlov** [comment]
Any volunteer?

**Comment by hdk5** [comment]
I'll do it

**Comment by asvetlov** [comment]
Fixed by #2955
